visvis2/material/_base.py

```diff
@@ -1,6 +1,97 @@
+from python_shader import Struct, vec2, mat4
+import numpy as np
+
+from .._wrappers import BufferWrapper
+
+
+# Definition of struct with standard info,
+# provided to each world-object as uniform at slot 0.
+stdinfo_type = Struct(
+    world_transform=mat4,
+    cam_transform=mat4,
+    projection_transform=mat4,
+    physical_size=vec2,
+    logical_size=vec2,
+)
```

Comment:
are we still allowing for non-python-shader based materials?

Reply:
Absolutely. This is just a convenient way to define the struct, and python-based shaders have the advantage that they can use this struct type in the signature.